docs/user-guide/docker/deploying-in-docker-compose.rst

```diff
@@ -127,106 +129,6 @@ corresponds to a :ref:`configured channel <docker-compose-configuring-channels>`
-
-      .. code-block:: bash
-
-          mkdir actions
-          mv actions.py actions/actions.py
-          touch actions/__init__.py  # the init file indicates actions.py is a python module
-          
-     The ``rasa/rasa-sdk`` image will automatically look for the actions in ``actions/actions.py``.
-
-  #. If your actions have any extra dependencies, create a list of them in a file,
-     ``actions/requirements-actions.txt``.
-  #. Create a file named ``Dockerfile`` in your project directory,
-     in which you'll extend the official SDK image, copy over your code, and add any custom dependencies (if necessary).
-     For example:
-
-      .. parsed-literal::
-
-         # Extend the official Rasa SDK image
-         FROM rasa/rasa-sdk:\ |version|.0
-
-         # Use subdirectory as working directory
-         WORKDIR /app
-
-         # Copy any additional custom requirements
-         COPY actions/requirements-actions.txt ./
-
-         # Change back to root user to install dependencies
-         USER root
-
-         # Install extra requirements for actions code, if necessary (otherwise comment this out)
-         RUN pip install -r requirements-actions.txt
-
-         # Copy actions folder to working directory
-         COPY ./actions /app/actions
-
-         # By best practices, don't run the code with root user
-         USER 1001
-
-You can then build the image via the following command:
-
-      .. code-block:: bash
-
-        docker build . -t <account_username>/<repository_name>:<custom_image_tag>
-
-The ``<custom_image_tag>`` should reference how this image will be different from others. For
-example, you could version or date your tags, as well as create different tags that have different code for production
-and development servers. You should create a new tag any time you update your code and want to re-deploy it.
-
-If you are using Docker Compose locally, you can use this image directly in your
-``docker-compose.yml``:
-
-      .. code-block:: yaml
-
-         version: '3.0'
-         services:
-           app:
-             image: <account_username>/<repository_name>:<custom_image_tag>
-
-If you're building this image to make it available from another server,
-for example a Rasa X or Rasa Enterprise deployment, you should push the image to a cloud repository.
-You can push the image to DockerHub via:
-
-      .. code-block:: bash
-
-        docker login --username <account_username> --password <account_password>
-        docker push <account_username>/<repository_name>:<custom_image_tag>
-
-To authenticate and push images to a different container registry, please refer to the documentation of
-your chosen container registry.
-
-Then, reference the new image tag in your ``docker-compose.override.yml``:
-
-      .. code-block:: yaml
-
-         version: '3.0'
-         services:
-           app:
-             image: <account_username>/<repository_name>:<custom_image_tag>
```

Comment:
is this still documented somewhere? since we got rid of it

Reply:
Yes, in step 2 https://rasa.com/docs/rasa-x/installation-and-setup/docker-compose-script/#connect-a-custom-action-server (instructions are different per install which is why it makes more sense anyway for "how to use it" to be listed in the deployment docs